libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -47,14 +66,16 @@ type CardColumn = CardDefinitionDirective[];
     selector: 'fd-fixed-card-layout',
     templateUrl: './fixed-card-layout.component.html',
     styleUrls: ['./fixed-card-layout.component.scss'],
-    changeDetection: ChangeDetectionStrategy.OnPush,
-    encapsulation: ViewEncapsulation.None
+    changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class FixedCardLayoutComponent implements OnInit, AfterContentInit, AfterViewInit, AfterViewChecked, OnDestroy {
     /** @hidden */
     @ContentChildren(CardDefinitionDirective)
     cards: QueryList<CardDefinitionDirective>;
 
+    @ViewChildren(FixedCardLayoutItemComponent)
```

Comment:
Let's add `/** @hidden */` comment, to hide it from docs

Reply:
fixed